### PR TITLE
create ai assignment history integration

### PR DESCRIPTION
**Pull request by SidelineCory24**
[BE PR](https://github.com/MetaPhase-Consulting/State-TalentMAP-API/pull/668)

- [x] Integrate with the new Assignment endpoint
- [x] Add status to assignment history for agenda item (Eg. EF, CP, AP)
- [x] Add additional information as appropriate (TOD description has been added as well)

### CODE REVIEW

**Comment by SidelineCory24** [comment]
Integrating with the new assignments endpoint has removed most the information displayed on the card. Should this be static content for now?

File: src/Components/ProfileDashboard/Assignments/AssignmentsListResultsCard/AssignmentsContent/AssignmentsContent.jsx [hunk 1]
```diff
@@ -1,6 +1,6 @@
 import { get } from 'lodash';
 // import { Link } from 'react-router-dom';
-import { NO_LANGUAGES, NO_POSITION_NUMBER, NO_POST, NO_SKILL } from 'Constants/SystemMessages';
+import { NO_LANGUAGES, NO_POSITION_NUMBER, NO_POST, NO_SKILL, NO_STATUS } from 'Constants/SystemMessages';
```
Comment (elizabeth-jimenez) [inline src/Components/ProfileDashboard/Assignments/AssignmentsListResultsCard/AssignmentsContent/AssignmentsContent.jsx:3]: This constant is used elsewhere to represent something else - i'd make just make a new one to represent what we want here

**Comment by SidelineCory24** [comment]
image

File: src/Components/Agenda/AgendaItemResearchPane/AgendaItemResearchPane.jsx [hunk 2]
```diff
@@ -49,8 +49,10 @@ const AgendaItemResearchPane = props => {
   const [selectedNav, setSelectedNav] = useState(get(tabs, '[0].value') || '');
 
   // assignments
-  const { data, error, loading /* , retry */ } = useDataLoader(api().get, `/fsbid/client/${perdet}/`);
-  const assignments = get(data, 'data.assignments') || [];
+  // eslint-disable-next-line max-len
+  // const { data, error, loading /* , retry */ } = useDataLoader(api().get, `/fsbid/client/${perdet}/`);
```
Comment (elizabeth-jimenez) [inline src/Components/Agenda/AgendaItemResearchPane/AgendaItemResearchPane.jsx:53]: ?

Comment (SidelineCory24) [reply src/Components/Agenda/AgendaItemResearchPane/AgendaItemResearchPane.jsx:53]: from original code, 🧼

File: src/Components/ProfileDashboard/Assignments/AssignmentsListResultsCard/AssignmentsContent/AssignmentsContent.jsx [hunk 3]
```diff
@@ -1,46 +1,70 @@
+/* eslint-disable max-len */
+/* eslint-disable no-unused-vars */
```
Comment (elizabeth-jimenez) [inline src/Components/ProfileDashboard/Assignments/AssignmentsListResultsCard/AssignmentsContent/AssignmentsContent.jsx:2]: ?

Comment (SidelineCory24) [reply src/Components/ProfileDashboard/Assignments/AssignmentsListResultsCard/AssignmentsContent/AssignmentsContent.jsx:2]: Commenting out some of the code until all of the data is coming through throws these lint errors. Is it better to do a single line disable instead an entire file?

File: src/Components/ProfileDashboard/Assignments/AssignmentsListResultsCard/AssignmentsContent/AssignmentsContent.jsx [hunk 4]
```diff
@@ -1,46 +1,70 @@
...
+            get(a, 'asg_pos_seq_num') ?
+              `(${get(a, 'asg_pos_seq_num')}) ` : NO_POSITION_NUMBER
+          }
+        </span>
+        {/* <Link to={`/archived/${get(assignment, 'position.position_id')}`}>View Position</Link> */}
+      </div>
+      <div>
+        <span className="bid-list-card-title-post">Location: </span>
+        {/* {getPostName(get(assignment, 'position.post', NO_POST))} */}
+        {'Hackneyville, Comoros'}
```
Comment (elizabeth-jimenez) [inline src/Components/ProfileDashboard/Assignments/AssignmentsListResultsCard/AssignmentsContent/AssignmentsContent.jsx:41]: can we change all the static ones to Coming Soon? That way it'll be easier to identify what needs to still be pulled in

File: src/Components/ProfileDashboard/Assignments/AssignmentsListResultsCard/AssignmentsContent/AssignmentsContent.jsx [hunk 5]
```diff
@@ -1,46 +1,70 @@
...
+          {
+            get(a, 'asg_pos_seq_num') ?
+              `(${get(a, 'asg_pos_seq_num')}) ` : NO_POSITION_NUMBER
+          }
+        </span>
+        {/* <Link to={`/archived/${get(assignment, 'position.position_id')}`}>View Position</Link> */}
+      </div>
+      <div>
+        <span className="bid-list-card-title-post">Location: </span>
+        {/* {getPostName(get(assignment, 'position.post', NO_POST))} */}
```
Comment (elizabeth-jimenez) [inline src/Components/ProfileDashboard/Assignments/AssignmentsListResultsCard/AssignmentsContent/AssignmentsContent.jsx:40]: i really like how you have these comments ready to go, for the integration 🎊

**Comment by elizabeth-jimenez** [comment]
This PR is compatible with existing Assignment History functionality

**Comment by elizabeth-jimenez** [comment]
image

**Comment by SidelineCory24** [comment]
closing this pr and opening another one for the rework

**Pull request closed without merging**